crates/ruma-state-res/benches/state_res_bench.rs

```diff
@@ -159,8 +175,12 @@ impl<E: Event> TestStore<E> {
     }
 
     /// Returns a Vec of the related auth events to the given `event`.
-    pub fn auth_event_ids(&self, room_id: &RoomId, event_ids: &[EventId]) -> Result<Vec<EventId>> {
-        let mut result = vec![];
+    pub fn auth_event_ids(
+        &self,
+        room_id: &RoomId,
+        event_ids: &[EventId],
+    ) -> Result<BTreeSet<EventId>> {
```

Comment:
Is this not a `HashSet` for a reason? Not a big deal either way.

Reply:
auth_events are usually just a few, so I skipped changing that as it should have almost no impact.